CMakeLists.txt

```diff
@@ -150,6 +150,9 @@ endif()
 # Kokkos recommendatation resulting in not using default GNU extensions
 set(CMAKE_CXX_EXTENSIONS OFF)
 
+# Build Kokkos as a shared library if building parthenon as a shared library
+SET (BUILD_SHARED_LIBS ON CACHE BOOL "Kokkos shared library option")
```

Comment:
I'm uncomfortable with this being set to `ON` no matter what. Although I'm not a `cmake` expert, so maybe this is fine.

Reply:
Behavior is now unchanged unless `CMAKE_INSTALL_PREFIX` is provided to cmake during configuration (determined here via `CMAKE_INSTALL_PREFIX_INITIALIZED_TO_DEFAULT`).